core/transformation/pathfinders/round_1/transform.py

```diff
@@ -59,18 +59,18 @@ def _submission_ref(
         submission_filename     - assigned during load_data
         data_blob               - includes "Sign Off Name", "Sign Off Role" and "Sign Off Date" from the transformed DF
     """
-    sign_off_name = df_dict["Sign off name"].iloc[0, 0]
-    sign_off_role = df_dict["Sign off role"].iloc[0, 0]
-    sign_off_date = df_dict["Sign off date"].iloc[0, 0].isoformat()
+    signatory_name = df_dict["Signatory name"].iloc[0, 0]
+    signatory_role = df_dict["Signatory role"].iloc[0, 0]
+    signature_date = df_dict["Signature date"].iloc[0, 0].isoformat()
     return create_dataframe(
         {
             "Submission Date": [datetime.now()],
             "Reporting Period Start": [PF_REPORTING_ROUND_TO_DATES[reporting_round]["start"]],
             "Reporting Period End": [PF_REPORTING_ROUND_TO_DATES[reporting_round]["end"]],
             "Reporting Round": [reporting_round],
-            "Sign Off Name": [sign_off_name],
-            "Sign Off Role": [sign_off_role],
-            "Sign Off Date": [sign_off_date],
+            "Sign Off Name": [signatory_name],
+            "Sign Off Role": [signatory_role],
+            "Sign Off Date": [signature_date],
```

Comment:
I think these column names are also in `mappings.py` as column names and will need updating there, but not sure if that's something that should be done as part of this PR or as part of _FMD-257 - Add missing PF columns to load_.

Reply:
Hey @MarcUsher, the column names used in `mappings.py` will be the ones outputted as part of the transformed DF here i.e., "Sign Off Name", "Sign Off Role" and "Sign Off Date", which have not been changed, so no change to `mappings.py` required